qa/tasks/ceph.py

```diff
@@ -1325,7 +1325,7 @@ def osd_scrub_pgs(ctx, config):
                     # request was missed.  do not do it every time because
                     # the scrub may be in progress or not reported yet and
                     # we will starve progress.
-                    manager.raw_cluster_cmd('pg', 'deep-scrub', pgid)
+                    manager.raw_cluster_cmd('tell', pgid, 'deep_scrub')
```

Comment:
Note: fixing this now to deep-scrub, as it should be

Reply:
Classic seems to use deep_scrub -- do you mean that you want to change it?  Or do you mean that the documentation has deep-scrub instead (if so, can you give me a link?)

Reply:
The documentation is solely for 'deep-scrub'. So is most of the implementation, apart from
where we've created a confusion when combining the implementations of the 'tell' and 'asok' paths.
I have a PR (pending approval) for fixing that.